test/webgl/context-state.spec.js

```diff
@@ -238,3 +240,54 @@ test('WebGLState#withParameters: recursive', t => {
 
   t.end();
 });
+
+// EXT_blend_minmax
+test('WebGLState#BlendEquationMinMax', t => {
+  const {gl, gl2} = fixture;
+  const contexts = {
+    ['WebGL1 Context']: gl,
+    ['WebGL2 Context']: gl2
+  };
+  const parametersArray = [
+    {
+      [GL.BLEND_EQUATION_RGB]: GL.MAX,
+      [GL.BLEND_EQUATION_ALPHA]: GL.MIN
+    },
+    {
+      blendEquation: GL.MAX
+    }
+  ];
+  const expectedArray = [
+    {
+      [GL.BLEND_EQUATION_RGB]: GL.MAX,
+      [GL.BLEND_EQUATION_ALPHA]: GL.MIN
+    },
+    {
+      [GL.BLEND_EQUATION_RGB]: GL.MAX,
+      [GL.BLEND_EQUATION_ALPHA]: GL.MAX
+    }
+  ];
+
+  for (const contextName in contexts) {
```

Comment:
I really like this test setup! 

I am however a little surprised that this works under Node.js as I suspect that the WebGL1 context provided by headless-gl does not support the extension. 

This makes me think we are getting a false positive. I suppose internally we just set the parameter without checking if it fails, and then just serve back the value from our cache, even though the true value is something else. 

To deal with this we probably would need an override of `gl.blendEquation` in the context polyfill.

Not sure that this complication is worth extra effort right now, we could address it in a future PR. Maybe just leave a comment in the test case about this.

Reply:
I will add a TODO, and keep #259 open.